test_pytest.py

```diff
@@ -84,3 +85,12 @@ def test_skill():
 
     response = app.test_client().get('/resume/skill')
     assert response.json[item_id] == example_skill
+
+
+def test_correct_spelling():
```

Comment:
Can we use the example from the issue here. Usually it's a good idea to have a test with the example that the issue mentions (if there's one)